androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/MainActivity.kt

```diff
@@ -9,17 +9,20 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
 import co.nimblehq.avishek.phong.kmmic.Greeting
+import co.nimblehq.avishek.phong.kmmic.android.ui.screen.login.SplashScreen
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            MyApplicationTheme {
+            ApplicationTheme {
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    GreetingView(Greeting().greet())
+                    //GreetingView(Greeting().greet())
```

Comment:
We can remove this view

Reply:
Resolved in acd211abce503054711d0c24171bd83cc8117768